account_operating_unit_sequence/models/account_payment.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    @api.model
+    def create(self, vals):
+        if vals.get("operating_unit_id", False):
+            ou = self.env["operating.unit"].browse(vals["operating_unit_id"])
+            if self.payment_type == 'transfer':
```

Comment:
Should this use vals['payment_type'] replace of self.payment_type ? because payment is still not created.

Reply:
Edited, Please test again.